doc/source/getting_started/accessing_models.rst

```diff
@@ -0,0 +1,109 @@
+.. _accessing_models:
+
+================
+Accessing Models
+================
+
+
+.. note:: Suppose you have started the Xinference server endpoint at ``http://0.0.0.0:8001``. 
+
+Please refer to :ref:`Launching Models <launching_models>` guide for get your model running.
+
+Using Clients
+=============
+
+
+LLM Model
+---------
+When the Ability of the LLM model includes "chat", we can converse with it using the model's chat interface:
+
+.. code-block:: python
+
+   from xinference.client import RESTfulClient
+   client = RESTfulClient("http://0.0.0.0:8001")
+   model = client.get_model(model_uid)
```

Comment:
Since the response below includes the model UID, I think it is better to get model with the model UID explicitly.

```suggestion
   model = client.get_model("f543d078-55f1-11ee-8fc0-0a28cc89f433")  # The model UID of the model you just launched.
```